client/app/containers/Blog/index.js

```diff
@@ -55,8 +56,11 @@ export class Blog extends Component {
 
   render() {
     const {
+      currentPage,
       focusedPost,
       loading,
+      maxPages,
+      onGetPosts,
       posts,
       routeParams,
     } = this.props;
```

Comment:
if you have to deconstruct inside render, can't you deconstruct as part of the render method?